src/content/docs/get-started/storefront-structure.mdx

```diff
@@ -13,7 +12,7 @@ import OptionsTable from '@components/OptionsTable.astro';
 
 The storefront project is a collection of **blocks** (Content components, widgets, and drop-in components), **scripts**, and **styles** designed for amazing performance, easy customizations, and rapid content development.
 
-## Big picture
+## Big picture :badge[Outdated]
```

Comment:
Not a fan of labelling something outdated . I understand you can't address everything at once, but I would prefer leaving it as is until you can get to it